runtime.md

```diff
@@ -50,10 +50,14 @@ If a hook returns a non-zero exit code, then an error is logged and the remainin
         "poststop": [
             {
                 "path": "/usr/sbin/cleanup.sh",
-                "args": ["-f"]
+                "args": ["cleanup.sh", "-f"]
             }
         ]
     }
 ```
 
-`path` is required for a hook. `args` and `env` are optional.
+`path` is required for a hook.
+`args` and `env` are optional.
+`args` includes the command as `args[0]`.
+If the `args` field is missing, empty, or `null`, the runtime will use `[path]`.
```

Comment:
I think we should use `[]` instead of `null`
`null` is confusing/misleading here, and `"args" : null` is invalided in the config.